Dockerfile

```diff
@@ -2,6 +2,8 @@
 ARG APP_IMAGE=hello-world:linux
 FROM $APP_IMAGE
 ADD https://github.com/microsoft/ApplicationInsights-Java/releases/download/3.1.1/applicationinsights-agent-3.1.1.jar /applicationinsights-agent.jar
+USER root
+RUN chmod 777 /applicationinsights-agent.jar
```

Comment:
not really good idea run as root

Reply:
as default root is the owner of the files added in docker...
with later version of docker it should be possible to define a different owner, but I should use only users defined in the /etc/passwd file... I will make a try...

Reply:
the use of the chown during the ADD worked :)